src/main/steps/c100-rebuild/hearing-without-notice/hearing-part1/template.njk

```diff
@@ -0,0 +1,16 @@
+{% extends "common/template.njk" %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+
+{% block page_content %}
+<h1 class="govuk-heading-xl govuk-!-margin-bottom-7">{{ title }}</h1>
+<div class="govuk-body-m">{{ line1 }}</div>
+{{ govukWarningText({
+
+text: "If you ask for a without notice hearing, the court may require you to attend on the same day as you submit your application. They will contact you to tell you when you need to go to the court.",
```

Comment:
@SangeethaPonnusamy Can you please check if we can move this warning text to content.ts file so that we can both English & Welsh translations ?

Reply:
I fixed this comment